velox/docs/functions/regexp.rst

```diff
@@ -33,3 +33,19 @@ See https://github.com/google/re2/wiki/Syntax for more information.
     the entire string by anchoring the pattern using ``^`` and ``$``::
 
         SELECT regexp_like('1a 2b 14m', '\d+b'); -- true
+
+.. function:: like(string, pattern) -> varchar
```

Comment:
Btw, I cannot seem to find "like" in presto docs:
https://prestodb.io/docs/current/functions/regexp.html
Is it hidden in another group of functions?
There is "regexp_like" though.

Reply:
LIKE is documented here: https://prestodb.io/docs/current/functions/comparison.html#like